jupyterlab_git/git.py

```diff
@@ -249,20 +249,24 @@ def __init__(self, contents_manager):
             }
 
         result = []
-        line_array = my_output.splitlines()
-        for line in line_array:
-            to1 = None
-            from_path = line[3:]
-            if line[0] == "R":
-                to0 = line[3:].split(" -> ")
-                to1 = to0[len(to0) - 1]
+        line_iterable = iter(my_output.strip("\x00").split('\x00'))
+        result = []
+        for line in line_iterable:
+            x = line[0]
+            y = line[1]
+            if line[0]=='R':
+                #If file was renamed then we need both this line
+                #and the next line, then we want to move onto the subsequent
+                #line. We can accomplish this by calling next on the iterable
+                to = line[3:]
+                from_path = next(line_iterable)
             else:
-                to1 = line[3:]
-            if to1.startswith('"'):
```

Comment:
why is this `"` handling no longer necessary?

Reply:
As far as I can tell git only adds the quotes when you have non-unicode characters and don't use `-z`

Reply:
If you're sure that's the case then I'm fine with it but if not then it's probably worth replacing `to` with `to.strip('\"')` to be sure